web-marketplace/src/lib/db/types/json-ld/project-metadata.ts

```diff
@@ -33,6 +33,7 @@ export interface AnchoredProjectMetadataBaseLD {
     'schema:url': string;
     'schema:creditText'?: string;
   };
+  marketType?: string[];
```

Comment:
this will come as regen:marketType in the jsonld formatting we support in the app

Reply:
see my suggestion for handling terrasos metadata type: https://github.com/regen-network/regen-web/pull/2527/commits/111a15b9abda1ed22f2944340d4a153f970f6a7b
TBD

Reply:
I'll update based on your changes when the other PR is merged.